.rubocop.yml

```diff
@@ -3,10 +3,11 @@ AllCops:
   Include:
     - "**/Gemfile"
     - "**/lib/**.rb"
+    - "**/lib/**/**.rb"
+    - "**/bin/**.rb"
     - "**/test/**.rb"
-    - "**/features/**/**.rb"
+    - "**/test/**/**.rb"
```

Comment:
Same as above, I wonder whether we can use `"**/test/**/*"` to cover lines 8 & 9 in one go.